firmware/CAN-DevBoard/CMakeLists.txt

```
@@ -42,8 +42,8 @@ set(PROJECT_INCLUDE_DIRECTORIES
     ${CMAKE_CURRENT_SOURCE_DIR}
     ${USER_DIR}/inc
     ${RESOURCES_DIR}/Drivers/SG_Drivers/CAN/Include
-    ${RESOURCES_DIR}/Drivers/SG_Drivers/Include)
-
+    ${RESOURCES_DIR}/Drivers/SG_Drivers/Include
+    ${RESOURCES_DIR}/Drivers/SG_Drivers/eeprom/inc)
```

```suggestion
    ${RESOURCES_DIR}/Drivers/SG_Drivers/Include)
```
get rid of separate folders in SG_Drivers (I will do CAN on my PR)